.github/prlint.json

```diff
@@ -1,8 +1,8 @@
 {
   "title": [
     {
-      "pattern": "^(feature|bug|minor|docs|DX):\\s.+\\S$",
-      "message": "Your title needs to match the convention: use '(feature|bug|minor|docs|DX):' prefix."
+      "pattern": "^(feat|bug|chore|CI|docs|DX|refactor|test)(\\([a-zA-Z0-9]+\\))?!?:\\s{1}\\S.+\\S$",
```

Comment:
looking on https://www.conventionalcommits.org/en/v1.0.0/ , I suggest we either use `feat|fix|...` (if we follow convention exactly), or we use `feature|bug|...` (if we don't)